test/end-to-end/README.md

```diff
@@ -49,6 +49,10 @@ Run end-to-end test like this:
 
 XUnit XML format result will be generated at */var/lib/docker/volumes/test-result-volume/_data/result.xml*
 
+## Test Suites and Cases Management
+
+The test suites and cases are managed by Google Sheets - [Welder-Web End-to-End Quality Testsuites](https://docs.google.com/a/redhat.com/spreadsheets/d/1ZYr8xLG-BNT6qMWVxlNSZEVELwo0rSEseA40oaIMNyw/edit?usp=sharing).
+
```

Comment:
Could you explain how this relates to the changes? And is it publicly visible? If not, it shouldn't be in the README